packages/angular/package.json

```diff
@@ -29,7 +29,7 @@
     "@coveo/search-token-server": "1.35.12",
     "@schematics/angular": "^14.0.1",
     "jest-config": "^27.4.7",
-    "typescript": "^4.3.5"
+    "typescript": "4.8.3"
```

Comment:
`@schematics/angular` is sneaky: it bundles typescript. If the version bundled and ours does not work nicely together 💥